src/hypothesis/strategies.py

```diff
@@ -174,16 +174,18 @@ def one_of(*args):
         except TypeError:
             pass
 
+    strategies = []
```

Comment:
Oh, heh. \__or\__ implicitly delegates to one\_of through a local circular import. I forgot it did that, so that's fine.

I do think it might be worth making the wrapper strategies branch, but I don't mind leaving that for now.